docs/api/clusterfile.md

```diff
@@ -0,0 +1,69 @@
+# Clusterfile definition
+
+```yaml
+apiVersion: sealer.aliyun.com/v1alpha1
+kind: Cluster
+metadata:
+  name: my-cluster
+  annotation:
+    sealer.aliyun.com/etcd: "/data/etcd" 
```

Comment:
I think we need tell user support field in clusterfile in different provider.

Reply:
A Clusterfile provider field that describes this.

Reply:
You mean that the concrete provider should provide their specific annotations?  does aliyun provides the annotations already? I think the annotations are little confising, let's add more descriptions?

Reply:
Some infra information will save in annotations like EIP, the "sealer.aliyun.com/etcd" key is not supported now, just an example